node/repo/fsrepo.go

```diff
@@ -185,27 +185,27 @@ func (worker) APIInfoEnvVars() (primary string, fallbacks []string, deprecated [
 	return "WORKER_API_INFO", nil, nil
 }
 
-type provider struct{}
+type curio struct{}
 
-var Provider provider
+var Curio curio
 
-func (provider) Type() string {
-	return "Provider"
+func (curio) Type() string {
+	return "Curio"
 }
 
-func (provider) Config() interface{} {
+func (curio) Config() interface{} {
 	return &struct{}{}
 }
 
-func (provider) APIFlags() []string {
-	return []string{"provider-api-url"}
+func (curio) APIFlags() []string {
+	return []string{"curio-api-url"}
 }
 
-func (provider) RepoFlags() []string {
-	return []string{"provider-repo"}
+func (curio) RepoFlags() []string {
+	return []string{"curio-repo"}
 }
 
-func (provider) APIInfoEnvVars() (primary string, fallbacks []string, deprecated []string) {
+func (curio) APIInfoEnvVars() (primary string, fallbacks []string, deprecated []string) {
 	return "PROVIDER_API_INFO", nil, nil
```

Comment:
oops?

Reply:
Yep!! Let me recheck other once more. I will add a new commit.

Reply:
Fixed the API_INFO. That was the only mistake. We should probably rebase `easyMigration` on master as Magik has pushed multiple new change with provider name.